common/src/test/java/org/opensearch/ml/common/transport/agent/MLAgentGetActionTest.java

```diff
@@ -0,0 +1,16 @@
+package org.opensearch.ml.common.transport.agent;
```

Comment:
please add license header

Reply:
It's showing resolved, but I don't see license header.

Reply:
added, thanks for checking!! I looks like ./gradlew spotlessApply didn't check the test packages. Added in the new commit